libs/effects/Effect/System.idr

```diff
@@ -14,18 +15,23 @@ data System : Effect where
      Time : sig System Integer
      GetEnv : String -> sig System (Maybe String)
      CSystem : String -> sig System Int
+     Usleep : (i : Int) ->
+              {auto prf : So (i >= fromInteger 0 && Delay (i <= fromInteger 1000000))} ->
```

Comment:
@artemohanjanyan Yeah, so I am not sure it is necessary for a constructor to have it. I would just take an argument `(inbounds : So (i >= 0 && i <= 1000000))`.